scripts/migrations/olids_2026/rewrite_downstream.py

```
@@ -0,0 +1,310 @@
+    for path in walk_files(scan):
+        rep = process_file(path, table_rules, cross_cutting_keys, args.dry_run)
+        if rep is None:
+            continue
+        if rep.renamed_counts:
+            touched.append(rep)
+        if rep.ambiguous_columns:
+            ambiguous.append(rep)
+        if rep.removed_hits:
+            removed_hits.append(rep)
+
+    print(f"=== Downstream rewrite summary ===\n")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)